### DISCUSSION TITLE
How do I resize a window while maintaining the aspect ratio?

### DISCUSSION THREAD

**Discussion by azizsayfiddin** [hyprwm/Hyprland#6031] [Q&A]
I want to resize the floating mpv window, but it doesn't resize in a fixed way. How can I do this?

**Comment by tharrmeehan** [comment]
bindm = $mainMod, mouse:273, resizewindow 1

If you want to bind floating windows with fixed aspect ratio to a key for example (for every window), alternatively you can use WindowRules (specifically the keepaspectratio one) to set it for mpv only.

https://wiki.hyprland.org/Configuring/Window-Rules/#rules
